src/imageproxy.rs

```diff
@@ -773,6 +773,19 @@ mod tests {
     use cap_std_ext::cap_std::fs::Dir;
     use rustix::fs::{memfd_create, MemfdFlags};
 
+    /// Check if we have skopeo
+    fn check_skopeo() -> bool {
+        static HAVE_SKOPEO: OnceLock<bool> = OnceLock::new();
+        *HAVE_SKOPEO.get_or_init(|| {
+            Command::new("skopeo")
```

Comment:
This seems broken since we allow overriding the skopeo command via the config (`config.skopeo_cmd`)... probably this function should also consider the config?